ext/opentelemetry-ext-stackdriver/src/opentelemetry/ext/stackdriver/utils.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2018, OpenCensus Authors
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import datetime
+import logging
+
+import opentelemetry.trace as trace_api
+from opentelemetry.sdk.util import ns_to_iso_str
+from opentelemetry.util import types
+
+# Max length is 128 bytes for a truncatable string.
+MAX_LENGTH = 128
+
+
+def get_truncatable_str(str_to_convert):
```

Comment:
Since all these helpers only apply to spans I think it'd be fine to keep them in the same module.